<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>253</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="reportSendButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Отправка отчета</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="continueButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Продолжить незаконченный отчет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newReport">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Новый отчет</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
